app/jobs/update_tokens_job.rb

```diff
@@ -1,9 +1,13 @@
+# frozen_string_literal: true
+
 class UpdateTokensJob < ApplicationJob
```

Comment:
```suggestion
class UpdateTokenJob < ApplicationJob
```

I think we are updating just one token at time, am I right :)?